### DISCUSSION TITLE
Jamulus OS in Cloud ?

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#1115] [Ideas]
Bonjour,

Can Jamulus OS be put in a Cloud (GCP, AWS, MS Azure, ...) instead of on an USB key ?
Or is there a special install version ?

Merci.

**Comment by ann0see** [comment]
I think this should be possible (just boot from the iso). I don't know about AWS etc. but maybe they allow iso upload.

**Comment by newlaurent62** [comment]
Jamulus OS is an Ubuntu studio distribution...with audio and x11 packages... 

While I've no experience with cloud, if your goal is to setup a Jamulus server, you might want to install a Linux server distribution on top of jamulus for cloud use.

**Comment by gilgongo** [comment]
@skrul posted this on the forums a while ago:

https://sourceforge.net/p/llcon/discussion/533517/thread/8f3f44458a/#752f

Public EC2 AMIs of Ubuntu 18.04 running Jamulus 3.5.0.

**Comment by pcar75** [comment]
@newlaurent62 

Why would I need to put a Linux server distribution on top of Jamulus OS ?

Note that our cloud use would be for a Jamulus-only private server with 5 or less musicians !

And what do you really mean by "on top of" (my coding and Unix days are way in the past) ?

**Comment by newlaurent62** [comment]
What I say is to use a linux server distribution and install/compile Jamulus software (and not Jamulus OS).

**Comment by gilgongo** [comment]
Yes, Jamulus OS is a desktop distro. Not very useful in the cloud I'm afraid. I think everyone using cloud servers is doing what @newlaurent62 says.

**Comment by pcar75** [comment]
Merci :-)

**Comment by pcar75** [comment]
Sorry, something is still nagging at me here !

Why would I need to put a server distribution instead of a user distribution to have a Jamulus server in the Cloud ?

It seems to me that the OS does not need to be multi-user, just multi-tasking   as only Jamulus sees other users in the form of UDP streams (like on a Jamulus private desktop server, or like someone using many links in a browser).  The only user that the OS sees would be me -the owner- for adjustements / maintenance ... !?

What am I missing ?

**Reply by npostavs** [reply]
> Why would I need to put a server distribution instead of a user distribution to have a Jamulus server in the Cloud ?

Typically a remote server has only a command line interface that you access via `ssh`. So the "user" parts (the graphical interface software) of the distribution are useless and just eat up your disk quota for nothing. You don't strictly *need* to use a "server" distribution though.

**Comment by gilgongo** [comment]
Hi @pcar75  We're trying to clear out the Issues into actionable to-dos now, so I hope you don'd mind if I move this to a Discussion. Thanks!
